frontend/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+
+function App() {
+  const [heading, setHeading] = useState("");
+
+  const handleClick = () => {
+    fetch('http://127.0.0.1:8000/test/')
```

Comment:
Create a env variable `BACKEND_BASE_URL` and an interceptor to add the base URL before each API call.
The API calls should be made like this `/test`.

Reply:
done in Feature/Scrapping PR